spark-submission-worker/src/main/java/org/apache/spark/k8s/operator/SparkAppSubmissionWorker.java

```diff
@@ -126,7 +126,7 @@ protected SparkAppDriverConf buildDriverConf(
     effectiveSparkConf.setIfMissing("spark.app.id", appId);
     return SparkAppDriverConf.create(
         effectiveSparkConf,
-        appId,
+        effectiveSparkConf.getAppId(),
```

Comment:
Please add a test case to `SparkAppSubmissionWorkerTest.java`.
- spark-submission-worker/src/test/java/org/apache/spark/k8s/operator/SparkAppSubmissionWorkerTest.java

Reply:
Thank you, @TQJADE .